src/graphql-result-size/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adeira/graphql-result-size",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "This module provides an implementation of an algorithm that calculates the (exact) size of GraphQL response objects without actually producing the response.",
   "homepage": "https://github.com/adeira/universe/tree/master/src/graphql-result-size",
```

Comment:
```suggestion
  "homepage": "https://github.com/adeira/graphql-result-size",
```

Reply:
Oh no, our monorepo scanner doesn't allow this change 🙈